### PR TITLE
add quantum serverless dashboard

### LINKED ISSUE #285: Grafana dashboard for system logs
### What is the expected behavior?

Implement grafana dashboard to display logs for system (keybernetes, ray, services (repository, gateway))

Epic https://github.com/Qiskit-Extensions/quantum-serverless/issues/174

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Summary
Fix #285 

### Details and comments

This PR add a grafana dashboard with repository and gateway services.

### CODE REVIEW

File: infrastructure/helm/quantumserverless/quantumserverlessdashboard-values.yaml [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    rayStartParams:
+      dashboard-host: '0.0.0.0'
+    ports: [{containerPort: 10001, name: client},{containerPort: 6379, name: redis},{containerPort: 8265, name: dashboard},{containerPort: 8080, name: metrics},{containerPort: 8000, name: serve},{containerPort: 4180, name: proxy}]
+    sidecarContainers:
+        - name: oauth-proxy
+          image: quay.io/oauth2-proxy/oauth2-proxy:v7.3.0
+          imagePullPolicy: IfNotPresent
+          args:
+          - --client-secret=CLIENTSECRET-CHANGEME
+          - --oidc-issuer-url=http://192.168.4.23:31059/realms/quantumserverless
```
Comment (psschwei) [inline infrastructure/helm/quantumserverless/quantumserverlessdashboard-values.yaml:116]:
```suggestion
          - --oidc-issuer-url=http://LOCAL-IP:31059/realms/quantumserverless
```

File: infrastructure/helm/quantumserverless/quantumserverlessdashboard-values.yaml [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+    - image: quay.io/gogatekeeper/gatekeeper:2.1.1
+      imagePullPolicy: IfNotPresent
+      name: gatekeeper
+      args:
+      - --no-redirects=true
+      - --forwarding-grant-type=client_credentials
+      - --listen=0.0.0.0:4180
+      - --client-id=rayapiserver
+      - --client-secret=APISERVERSECRET-CHANGEME
+      - --discovery-url=http://192.168.4.23:31059/realms/quantumserverless
```
Comment (psschwei) [inline infrastructure/helm/quantumserverless/quantumserverlessdashboard-values.yaml:249]:
```suggestion
      - --discovery-url=http://LOCAL-IP:31059/realms/quantumserverless
```

File: infrastructure/helm/quantumserverless/quantumserverlessdashboard-values.yaml [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+          - --client-secret=CLIENTSECRET-CHANGEME
+          - --oidc-issuer-url=http://192.168.4.23:31059/realms/quantumserverless
+          - --oidc-extra-audience="account"
+          - --email-domain="*"
+          - --insecure-oidc-allow-unverified-email=true
+          - --http-address=0.0.0.0:4180
+          - --cookie-secret=SECRET0123456789
+          - --provider=keycloak-oidc
+          - --client-id=rayclient
+          - --upstream="http://quantumserverlessdashboard-kuberay-head-svc:8265"
```
Comment (psschwei) [inline infrastructure/helm/quantumserverless/quantumserverlessdashboard-values.yaml:124]: Should this use `HELM-RELEASE` ?

**Review by psschwei — COMMENTED** [review]
There were a couple of hardcoded values that were variable in the default values.yaml ... 

Also, assuming that we need to apply the quantumserverlessdashboard-values.yaml file in order to see the new dashboards (?)

**Comment by akihikokuroda** [comment]
@psschwei It's my bad.  I pushed completely wrong file. Sorry.

**Review by psschwei — APPROVED** [review]

**Pull request merged**